kbase-extension/static/kbase/js/util/jobLogViewer.js

```diff
@@ -417,18 +428,21 @@ define([
             requestLoop = null, // the timeout object
             scrollToEndOnNext = false;
 
-        // VIEW ACTIONS
+        function getLogPanel() {
+            return ui.getElement('log-panel');
+        }
 
-        function scheduleNextRequest() {
+        // VIEW ACTIONS
+        function scheduleLogRequest() {
```

Comment:
changed the method names to include what the request is for -- status or log -- to make life easier for developers